src/components/MessageList.js

```diff
@@ -144,6 +150,7 @@ class MessageList extends PureComponent {
 
   static defaultProps = {
     Message: MessageSimple,
+    MessageSystem: EventComponent,
```

Comment:
`MessageSystem` seems more appropriate. Should change EventComponent to MessageSystem as well later. Not touching currently to avoid breaking styles and stuff